server/game/core/ability/PlayerOrCardAbility.js

```diff
@@ -54,6 +55,8 @@ class PlayerOrCardAbility {
         this.keyword = null;
         this.type = type;
         this.optional = !!properties.optional;
+        //Nested abilities rule(Comp Rules 2.0 7.6.11): triggers should always resolve triggers triggered during their effects right after their effects
+        this.resolveTriggersAfter = this.type === AbilityType.Triggered || !!properties.resolveTriggersAfter;
```

Comment:
I don't think we can really do a proper test of nesting until on-defeat timing is working correctly, sadly. Could you just leave a TODO here though that we need to test this?